resources/views/components/prose/index.blade.php

```diff
@@ -0,0 +1,11 @@
+{{--
+This is the default prose, suitable for SEO content or other text.
```

Comment:
I think it's a good idea to add some info why we're not using all the classes here. So why do we separate it into a CSS file? While Tailwind doesn't recommend it: https://tailwindcss.com/docs/reusing-styles